app/info_html.R

```diff
@@ -134,7 +134,16 @@ renderMethodsHtml <- function() {
                adds 1 direct contact to the person's risk level." )
     ), # end of ol
     tags$p(""),
-    tags$p("We'll be doing our best to update these assumptions as additional knowledge about the virus becomes available.")
+    #tags$p("We'll be doing our best to update these assumptions as additional knowledge about the virus becomes available."),
+    tags$h4("In the Works [or Features/Updates to Come]"),
```

Comment:
Emma, great section! Do you mind rewording this title? Would "In the works" suffice, or do we need the contents within the square bracket?

Reply:
Thank you Cindy for your feedback.  Yes "In the works" I think is a good title. I added the square brackets as other possible titles. I will remove those options.